docs/nufft.rst

```diff
@@ -166,6 +166,57 @@ Where :math:`E_mn = e^i\Delta\omega_0(u_n)t_m`.
 
 .. _nufft-algo:
 
+Subspace Projection Model
+~~~~~~~~~~~~~~~~~~~~~~~~~
+In several MRI applications, such as dynamic or quantitative MRI, a single acquisition provides a stack of images, each representing a single time frame (for dynamic MRI) or a single contrast (for quantitative MRI).
```

Comment:
it should be clarified whether you handle images or volumes (resp., 2D or 3D) as it matters in regards to the definition of under-sampling patterns, i.e. whether they're composed of multishot trajectories (as in 3D) or of single shots (as in 2D like in the seminal MRF paper)